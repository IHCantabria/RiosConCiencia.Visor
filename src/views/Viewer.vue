<template>
  <div class="view-viewer grid">
    <app-map-viewer class="map"></app-map-viewer>
    <app-panels></app-panels>
    <app-fetch-manager></app-fetch-manager>
  </div>
</template>
<script>
import AppPanels from "@/components/layout/AppPanels.vue";
import AppMapViewer from "@/components/layout/AppMapViewer.vue";
import AppFetchManager from "@/components/layout/AppFetchManager.vue";
export default {
  components: {
    "app-map-viewer": AppMapViewer,
    "app-panels": AppPanels,
    "app-fetch-manager": AppFetchManager
  }
};
</script>
<style src="@fortawesome/fontawesome-free/css/all.min.css"></style>
<style scoped lang="scss">
.view-viewer {
  height: 100%;
}
</style>
