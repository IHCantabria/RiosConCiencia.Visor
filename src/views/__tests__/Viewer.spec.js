import { shallowMount } from "@vue/test-utils";
import Viewer from "../Viewer.vue";
import merge from "lodash.merge";

function createWrapper(overrides) {
  const defaultMountingOptions = {};
  return shallowMount(Viewer, merge(defaultMountingOptions, overrides));
}

describe("viewer.vue", () => {
  //SNAPSHOT TESTS
  it("renders correctly", () => {
    expect.assertions(1);
    const wrapper = createWrapper();
    expect(wrapper.element).toMatchSnapshot();
  });
});
