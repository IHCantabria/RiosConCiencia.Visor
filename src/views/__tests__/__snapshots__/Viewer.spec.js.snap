// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`viewer.vue renders correctly 1`] = `
<div
  class="view-viewer grid"
>
  <app-map-viewer-stub
    class="map"
  />
   
  <app-panels-stub />
   
  <app-fetch-manager-stub />
</div>
`;
